### ISSUE TITLE
Javascript bookmarklet multiplication

### ISSUE THREAD

**Issue by danpeig** [floccusaddon/floccus#1731]
### Which version of floccus are you using?

5.3.0

### How many bookmarks do you have, roughly?

200

### Are you using other means to sync bookmarks in parallel to floccus?

No

### Sync method

Nextcloud Bookmarks

### Which browser are you using? In case you are using the phone App, specify the Android or iOS version and device please.

Chrome 129 and Firefox 130

### Which version of Nextcloud Bookmarks are you using? (if relevant)

14.2.6

### Which version of Nextcloud? (if relevant)

29.0.7

### Describe the Bug

Duplication of javascript bookmarklets after reopening the browser.

Every time the browser is reopened and a "pull down" operation is executed, a new bookmarklet will be added. If the "pull down" operation is executed more than once during the same browsing session, there is no duplication.

**Bookmarklet**
```javascript
javascript:(function(){var a=window,b=document,c=encodeURIComponent,e=c(document.title),d=a.open('https://nextcloud.myserver.com/apps/bookmarks/bookmarklet?url='+c(b.location)+'&title='+e,'bkmk_popup','left='+((a.screenX||a.screenLeft)+10)+',top='+((a.screenY||a.screenTop)+10)+',height=650px,width=550px,resizable=1,alwaysRaised=1');a.setTimeout(function(){d.focus()},300);})();
```

### Expected Behavior

The javascript bookmarklet shoud not be multiplied.

### To Reproduce

1. From Nextcloud bookmarks, create a bookmarklet (for adding new bookmarks)  and save it in the browser bookmark list.
2. **Push up** the bookmarklet so it appears in Nextcloud online bookmarks list
3. **Pull down** using Floccus extension in another browser (Firefox or Chrome). The bookmarklet should appear in the new browser.
4. Reopen the browser and **pull down** again. Now there are multiple bookmarklets. If you **pull down** within the same session, no extra bookmarklets will appear. The duplication only occurs when a new browser session is initiated.

**NOTE:** Sync is not enabled. "Push up" and "pull down" operations are executed manually.

### Debug log provided

- [ ] I have provided a debug log file

**Comment by marcelklehr** [comment]
Hi @danpeig 
I was able to reproduce this! :tada: Thank you for your excellent steps to reproduce!

**Comment by danpeig** [comment]
Great! If you need more information, logs or tests, please let me know.

**Comment by canufarm** [comment]
Hi,

I have a very similiar issue: When I open Firefox, start a manual or auto sync of floccus, then all my bookmarklets in the nextcloud bookmark app are removed to the trash of the bookmark app in Nextcloud. If I then restore them from the trash one by one and do some other syncs with floccus, nothing will be deleted any more in this session. But after closing and reopening Firefox the same issue appears each an every time. Only bookmarklets are deleted/put into trash in the nextcloud app, not any other bookmarks.

For now I stopped the sync or only make an upload from Firefox to Nextcloud to not miss any bookmarklets. 

Any help appreciated! Thanks.

**Comment by marcelklehr** [comment]
Hi everyone,

I've just released v5.3.1 (currently in Open Testing on Google play), which should fix this. Can you please confirm?

**Comment by danpeig** [comment]
Confirming the fix is working as expected:
- Firefox Windows: OK
- Firefox Linux: OK
- Chrome Windows: OK
- Edge Windows: OK

**Comment by Nebucatnetzer** [comment]
I'm seeing this problem again in my setup with Firefox.

**Comment by marcelklehr** [comment]
@Nebucatnetzer Could you please open a new issue?
